llvm/lib/ToolDrivers/llvm-lib/Options.td

```diff
@@ -28,6 +28,11 @@ def nativedeffile : P<"defArm64Native", "def file to use to generate native ARM6
 def llvmlibthin : F<"llvmlibthin">,
     HelpText<"Make .lib point to .obj files instead of copying their contents">;
 
+def llvmlibindex : F<"llvmlibindex">,
+    HelpText<"Write an index to the output (default)">;
+def llvmlibindex_no : F<"llvmlibindex:no">,
+    HelpText<"Do not write an index to the output">;
```

Comment:
These could be a `B` style option (check out e.g. `allowbind` for an example).

Reply:
Done, thanks! I thought there's no B<> in this file, but there is (and if there wasn't, I should've added one).

(There's no allowbind in this file at least :P But there is WX.)